### PR TITLE
Supported Kernels

### PR DESCRIPTION
**Pull request by kmunson1973**
Added supported kernels page.

### CODE REVIEW

File: content/reference/knowledge-base/supported-kernels.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+```text
+yum install kernel-headers-`uname -r`  
+yum install kernel-devel-`uname -r`
+
+```
+
+#### Ubuntu
+
+```text
+apt linux-headers-$(uname -r)
```
Comment (kschoche) [inline content/reference/knowledge-base/supported-kernels.md]: apt install?

File: content/reference/knowledge-base/supported-kernels.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+#### Ubuntu
+
+```text
+apt linux-headers-$(uname -r)  
+
+```
+
+#### Portworx CentOS/RHEL supported kernels
```
Comment (kschoche) [inline content/reference/knowledge-base/supported-kernels.md]: Is this the list of prebuilt kernels or the supported kernels we can build against?

**Comment by harsh-px** [comment]
Closing this one as #50 has these changes.

**Comment by harsh-px** [comment]
Closing this one as #50 has these changes.

**Comment by harsh-px** [comment]
#50 supercedes this. Closing this one.

**Pull request closed without merging**